### PR TITLE
addition of modified protocol methods

### PR DESCRIPTION
**Pull request by malloc3**

### CODE REVIEW

**Comment by malloc3** [comment]
@dvnstrcklnd

File: microtiter_plates/libraries/platelayoutgenerator/source.rb [hunk 1]
```diff
@@ -35,6 +42,16 @@ def next_group(column: nil)
     @layout.slice!(i, @group_size)
   end
 
+  def iterate_column(column)
+    return nil if column.nil?
```
Comment (dvnstrcklnd) [inline microtiter_plates/libraries/platelayoutgenerator/source.rb]: @column is initialized as []. Is there anywhere it can be set to nil?

Comment (dvnstrcklnd) [reply microtiter_plates/libraries/platelayoutgenerator/source.rb]: Use `column.blank?` instead. (I think this is a Rails method, so may not show up in the Ruby API)

**Comment by dvnstrcklnd** [comment]
Seems mostly good. A few issues.

**Pull request merged**